### DISCUSSION TITLE
Is a GUI planned in the future

### DISCUSSION THREAD

**Discussion by katastrophe92** [espanso/espanso#1838] [Feature requests and Ideas]
First of all thank you Espanso team for this amazing repo! I can use it for what I need but almost every time I need to see the documentation when I need to make complicated snippets. A GUI whit infos in my opinion would be awesome and easier. Im sure that many users can't bother to use a software without a GUI. Have a good day!

**Comment by smeech** [comment]
There are several repos [available,](https://github.com/search?q=espanso%20gui&type=repositories) but I haven't had a chance to try any of them. I'm not aware of any development in this direction in the main project, however, although it's a much requested feature.

**Comment by katastrophe92** [comment]
> There are several repos [available,](https://github.com/search?q=espanso%20gui&type=repositories) but I haven't had a chance to try any of them. I'm not aware of any development in this direction in the main project, however, although it's a much requested feature.

Hi! I know about them but I wish one from the Espanso developer 😝
